browser/src/control/Control.TopToolbar.js

```diff
@@ -434,9 +435,24 @@ L.Control.TopToolbar = L.Control.extend({
 			w2ui['editbar'].hide('savebreak');
 		}
 
-		if (e.EnableInsertRemoteImage === true && w2ui['editbar']) {
-			w2ui['editbar'].hide('insertgraphic');
-			w2ui['editbar'].show('menugraphic');
+		if (w2ui['editbar']) {
+			if (e.EnableInsertRemoteImage === true && !e.DisableInsertLocalImage) {
+				w2ui['editbar'].hide('insertgraphic');
+				w2ui['editbar'].hide('insertremotegraphic');
```

Comment:
it seems odd to have something called "EnableInsertRemoveImage" hiding something called "insertremovegraphic"?

Reply:
I can rename it `EnableInsertLocalImage` and have it set to `true` by default. I thought `false` by default was fine, but either work.

Reply:
but it shows menugraphic below. If we have the two options local and remote then we show the menu, otherwise we show one or the other.

Previously you couldn't disable the local image insertion, so there was only two choices. remoteimage enabled -> menu, otherwise just the button. The new button is added above (line 230).